### PR TITLE
Create gcs-inventory-loader.ipynb

### PR DESCRIPTION
**Pull request by Marianie-Simeon**
Created a motebook that uses gcs-inventory-loader

### CODE REVIEW

File: scripts/gcs-inventory-loader.ipynb [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# GCS Inventory Loader Introduction\n",
+    "Load your GCS bucket inventory into BigQuery (or stdout) fast with this tool.\n",
+    "\n",
+    "It can be very useful to have an inventory of your GCS objects and their metadata, particularly in a powerful database like BigQuery. The GCS listing API supports filtering by prefixes, but more complex queries can't be done via API. Using a database, you can find out lots of information about the data you have in GCS, such as finding very large objects, very old or stale objects, etc.\n",
+    "\n",
+    "This utility will help you bulk load an object listing to stdout, or directly into BigQuery. It can also help you keep your inventory up-to-date with the listen command.\n",
+    "\n",
```
Comment (schaluva) [inline scripts/gcs-inventory-loader.ipynb:13]: The introduction is great! However, if I had never used it before I think it is a tiny bit confusing what is actually happening in the notebook so maybe a summary section like:
1. Set up the Notebook
2. Load your workspace bucket to a BQ project
3. Visualize object metadata


Also maybe a note or two about how they have to come to this notebook with a BQ project and dataset already created.

File: scripts/gcs-inventory-loader.ipynb [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+   "metadata": {},
+   "source": [
+    "# Configure the default.cfg file\n",
+    "Configure the .default.cfg file to point to the right BQ project for the resulting dataset and also the project within which the buckets of interest live\n",
+    "\n",
+    "Change These Values:\n",
+    "- PROJECT=The_BigQuery_Project\n",
+    "- GCS_PROJECT=The_Bucket_Project\n",
+    "- DATASET_NAME=The_BigQuery_Dataset\n",
+    "- INVENTORY_TABLE=The_BigQuery_Table\n",
```
[the remark this answers is not in the document]
Comment (schaluva) [reply scripts/gcs-inventory-loader.ipynb:69]: Also - a few more words to define the variables. For example `"INVENTORY_TABLE" is what they want to call the resulting table of gcs-inventory-loader where the object metadata will be uploaded`

**Review by schaluva — CHANGES_REQUESTED** [review]

File: scripts/gcs-inventory-loader.ipynb [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+    "data_timeline_table"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
```
Comment (mmorgantaylor) [inline scripts/gcs-inventory-loader.ipynb:443]: you only need to run the `%matplotlib inline` command once - you've already run it up above, no need to re-run here (can delete this cell)

**Comment by mmorgantaylor** [comment]
i'll go look it up from the ticket now, but for PRs that involve notebooks, could you in future put the Terra version of the notebook into the PR description? that would make reviewing it much easier!!

**Pull request closed without merging**